WASM.md

```diff
@@ -1,23 +1,68 @@
-Proof of Concept WebAssembly Support.
+WebAssembly Extension Support
 
-# WebAssembly VM
+# Build Dependencies
 
-The WebAssembly VM is https://github.com/WAVM/WAVM. This VM was chosen for the
-proof of concept because:
+## Install glib2.0 if required (on Debian/Ubuntu)
 
-  - It is OSS with a permissive license.
-  - It compiles with LLVM for high performance.
-  - It can attach pre-compiled code to the WASM files (e.g. during distribution
-      from a central controller) for faster startup and reduced load on a pool
-      of proxies.
+apt-get install libglib2.0-dev
 
-# TODO
+## emscripten
```

Comment:
`emscripten` is a dependency for regenerating test modules, isn't it?

Reply:
It is also a dependency for building your own WebAssembly module, so it is should be in the main section.